command/project-create.go

```diff
@@ -35,6 +35,23 @@ func (c *ProjectCreateCommand) Run(args []string) int {
 	dbEngine := c.flagSet.String("db-engine", "", "Select database engine")
 	dbSize := c.flagSet.String("db-size", "", "Select database size")
 	dbVersion := c.flagSet.String("db-version", "", "Select database version")
+
+	consulEnabled := c.flagSet.Bool("consul", false, "Enable Consul")
```

Comment:
Like most other "features" have a `feat_enabled` pattern, I think that same should apply for consul nomad and vault

Reply:
The only instance of such case i can find is `backup-enabled` in db-set.go 
Should i rename my flag to something in the like of `consul-enabled` ?

thanks for your insight :)